miner/payload_building_test.go

```diff
@@ -283,13 +307,21 @@ func testBuildPayload(t *testing.T, noTxPool, interrupt bool, params1559 []byte)
 	// make sure the 1559 params we've specied (if any) ends up in both the full and empty block headers
 	var expected []byte
 	if len(params1559) != 0 {
-		expected = []byte{0}
+		versionByte := byte(0)
+		if config.IsConfigurableMinBaseFee(testTimestamp) {
+			versionByte = 1
+		}
+		expected = []byte{versionByte}
+
 		d, _ := eip1559.DecodeHolocene1559Params(params1559)
 		if d == 0 {
 			expected = append(expected, eip1559.EncodeHolocene1559Params(250, 6)...) // canyon defaults
 		} else {
 			expected = append(expected, params1559...)
 		}
+		if versionByte == 1 {
+			expected = append(expected, byte(minBaseFeeLog2))
```

Comment:
FYI the CI is failing linting on this line:

miner/payload_building_test.go:323:36: unnecessary conversion (unconvert)
                        expected = append(expected, byte(minBaseFeeLog2))
                        
```suggestion
			expected = append(expected, minBaseFeeLog2)
```